### PR TITLE
Adding on-boarding guide for multimeter

### PR DESCRIPTION
**Pull request by yatri1609**
First steps to issue #972 

**Changes**:
Added on-boarding guide for the multimeter.

**Screenshot/s for the changes**: 
screenshot_2018-06-16-11-36-14-011_org fossasia pslab
screenshot_2018-06-16-11-36-20-569_org fossasia pslab
screenshot_2018-06-16-11-36-25-653_org fossasia pslab
screenshot_2018-06-16-11-36-29-919_org fossasia pslab

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [ ] I have requested reviews from other members

**APK for testing**: 
[app-debug.zip](https://github.com/fossasia/pslab-android/files/2107907/app-debug.zip)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -7,7 +7,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.0'
+        classpath 'com.android.tools.build:gradle:3.1.2'
```
Comment (CloudyPadmal) [inline build.gradle]: Please revert this change

**Comment by yatri1609** [comment]
Ya I'll do it !

File: build.gradle (hunk 1 above)
Comment (yatri1609) [reply build.gradle]: Ya I'll do it.

**Comment by yatri1609** [comment]
[app-debug.zip](https://github.com/fossasia/pslab-android/files/2108074/app-debug.zip)

This is the updated apk

**Comment by yatri1609** [comment]
@CloudyPadmal can you please have a look at updated files ?

**Comment by CloudyPadmal** [comment]
@harsh-2711 @abhinavraj23 @Avjeet @Vikum94 we need to discuss about this and come to a conclusion if we are going to use dialog boxes or bottom sheet guides.

**Comment by yatri1609** [comment]
I think we should go with the bottomsheet as we discussed so should we close this PR? @CloudyPadmal

**Comment by abhinavraj23** [comment]
Yes @yatri1609 I think closing this PR should be okay as we are deciding to go with bottomsheet guide :+1:
